feathr_project/feathr/version.py

```diff
@@ -1 +1,10 @@
-__version__ = "0.9.0-rc2"
\ No newline at end of file
+__version__ = "0.9.0-rc2"
+
+def get_version():
+    return __version__
+
+# Decouple Feathr MAVEN Version from Feathr Python SDK Version
+import os
+def get_maven_artifact():
+    maven_version = os.environ.get("FEATHR_MAVEN_VERSION", __version__)
+    return f"com.linkedin.feathr:feathr_2.12:{maven_version}"
```

Comment:
Suggest following name change to make code more readable
- maven_version => maven_artifact_version 
- get_maven_artifact => get_maven_artifact_fullname
- FEATHR_MAVEN_VERSION =>  MAVEN_ARTIFACT_VERSION